aa_engine/aas_retrieve_inputs_part2.m

```diff
@@ -86,14 +85,17 @@
                                     numtotransfer=numtotransfer+1;
                                 end;
                                 if (streamfiles(depind).wasnamechange)
-                                    aas_copyfromremote(aap, streamfiles(depind).inputstream.host, fullfile(streamfiles(depind).src,streamfiles(depind).fns{ind}),streamfiles(depind).fns_dest_full{ind},'verbose',0,'allowcache',streamfiles(depind).inputstream.allowcache);
+                                    aas_copyfromremote(aap, streamfiles(depind).inputstream.host, fullfile(streamfiles(depind).src,streamfiles(depind).fns{ind}),streamfiles(depind).fns_dest_full{ind},'verbose',0,'allowcache',streamfiles(depind).inputstream.allowcache,'allowremotesymlinks',use_remotesymlinks);
```

Comment:
@tiborauer I will suggest adding an aap parameter such as "remotecpverbose" to use instead of hardcoding the value of verbose here. However, I am not sure which parameter set I should set the default to so that other pipelines do not break due to the lack of a value. I am assuming [aa_parametersets/aap_parameters_defaults.xml](https://github.com/automaticanalysis/automaticanalysis/pull/372/files#diff-e28b032449832ae153e74104b1b027ad796f18daccd49f04691f3e8f4be5bd90) would be the correct one?